.github/workflows/ruby.yml

```diff
@@ -0,0 +1,117 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+name: Ruby
+
+on:
+  push:
+    branches: [ production ]
+  pull_request:
+    branches: [ production ]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    services:
+      db:
+        image: postgres:11
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5.0
+      - run: | 
+          bundle install --without production development
+      - name: Build and run tests
+        env:
+          RAILS_ENV: test
+          PGHOST: localhost
+          PGPASSWORD : postgres
+        run: |
+          psql -c 'create database kontes_terbuka_test;' -U postgres
+          cp config/database.yml.travis config/database.yml
+          cp .env.default .env
+          bundle exec rake db:structure:load
+          bundle exec rake db:setup
```

Comment:
If you're running `db:setup` I don't think you need the `create database` command and the `db:structure:load` command